### PR TITLE
init postgres docker

### PR DESCRIPTION
**Pull request by agustinpurita**
## Pull request type

- [x] Refactoring (no functional changes, no api changes)

### CODE REVIEW

File: docker_postgres_init.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE USER docker WITH PASSWORD 'password' CREATEDB;
```
Comment (martinjaimem) [inline docker_postgres_init.sql:1]: Just in case, I think Postgres creates a user/pass `postgres` `postgres` by default

File: .example.env.test [hunk 2]
```diff
@@ -5,7 +5,7 @@ TYPEORM_TYPE = postgres
 TYPEORM_HOST = localhost
 TYPEORM_USERNAME = 
 TYPEORM_PASSWORD = 
-TYPEORM_DATABASE = express-api-base-test
+TYPEORM_DATABASE = eexpress_api_base_test
```
Comment (martinjaimem) [inline .example.env.test]:
```suggestion
TYPEORM_DATABASE = express_api_base_test
```

**Review by martinjaimem — COMMENTED** [review]
LGTM 👍

File: docker_postgres_init.sql (hunk 1 above)
Comment (agustinpurita) [reply docker_postgres_init.sql:1]: Correct! Postgres creates a user/pass as default to connect, but i think is a good practice creates a new user as owner of databases.
If you look, I did not modify the user/pass to connect with postgres service

File: .example.env.test (hunk 2 above)
Comment (agustinpurita) [reply .example.env.test]: Fixed, thanks!

**Review by martinjaimem — APPROVED** [review]

**Review by Alxzu — APPROVED** [review]
Nice!

**Pull request merged**